src/Connections/GithubRepo.ts

```diff
@@ -1258,6 +1261,48 @@ export class GitHubRepoConnection extends CommandConnection<GitHubRepoConnection
             },
         }
     }
+    
+    public static async searchInstallationForRepos(octokit: Octokit, orgName: string, installationId: number, searchTerms?: string) {
+        // First, do a search on GitHub for repos. This will use the user's context so it will find user repos.
+        let searchRepos: string[]|null = null;
+        if (searchTerms) {
+            const terms = encodeURIComponent(searchTerms);
+            const searchResultsData = (await octokit.search.repos({
+                q: `${terms} org:${orgName} `,
```

Comment:
Could any user abuse `term` to query terms in other orgs with Hookshot's GitHub token?
e.g. `term = "hl3 org:valve"`

No, because it's queried by the user's webbrowser, right?

Reply:
The users access token yeah.